cluster-scope/base/core/namespaces/acm/namespace.yaml

```diff
@@ -4,6 +4,8 @@ kind: Namespace
 metadata:
   annotations:
     openshift.io/display-name: "Operate First: ACM"
-    openshift.io/requester: operate-first
-  name: acm
+  labels:
+    op1st-project-owner: operate-first
+    op1st-onboarding-issue: 24
+  name: https://www.operate-first.cloud/apps/content/acm/README.html
```

Comment:
>    name: https://www.operate-first.cloud/apps/content/acm/README.html 

here you are changing the resource's name not the label

Reply:
Thanks for the catch!